src/api/routes/users.ts

```diff
@@ -247,6 +247,29 @@ const listPriviledgedUserDataAccess = procedure
   });
 });
 
+const remove = procedure
+  .use(authUser())
+  .input(z.object({
+    id: z.string(),
+  }))
+  .mutation(async ({ input, ctx }) => 
+{
+  const isUserOrPRManager = isPermitted(ctx.user.roles, ['UserManager', 'PrivilegedRoleManager']);
```

Comment:
1. only UserManager is necessary.
2. use `authUser()` to simplify the logic. so:

```
  .user(authUser("UserManager"))
```

and remove this text.